apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx

```diff
@@ -51,14 +51,15 @@ Learn how to add custom integration tests in the following steps.
   }
 ```
 
----
-
 ## Step 2: Integrating custom Cypress tests into your FastStore project
 
+Just like any other project using Cypress, all standard Cypress folders and configuration files are supported. (see official Cypress configuration guide [here](https://docs.cypress.io/guides/references/configuration))
+
 1. In the root directory of your storefront project, create a new folder named `cypress`.
-2. Add your custom integration tests within the `cypress` folder. 
-3. Open a pull request with the changes you have made.
-4. Within the pull request, navigate to **Checks > Integration Tests** and check the **Details** section.
+2. Add your custom integration tests within the `cypress/integration` folder.
+3. With the local server running, run `yarn test` to open the Cypress app and to check your tests execution.
+4. Open a pull request with the changes you have made.
+5. Within the pull request, navigate to **Checks > Integration Tests** and check the **Details** section.
 
 ![checks-integration-tests](https://vtexhelp.vtexassets.com/assets/docs/src/custom-test-checks___583ac9b78e43ebdafde93a735a47d61b.png)
 
```

Comment:
I think the last step (_Review the results to check the execution status of your custom integration tests._) was supposed to be number "6".